src/styles/screenForgotPassword.js

```diff
@@ -0,0 +1,61 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+    container: {
+        backgroundColor: '#001435',
+        flex: 1,
+        paddingVertical: 30
+    },
+    viewLogo: {
+        alignItems: 'center'
+    },
+    logo: {
+        height: 121,
+        width: 220
+    },
+    viewTop: {
+        justifyContent: 'center',
+        paddingTop: 10
+    },
+    textnewPassword: {
+        color: '#FFFFFF',
+        textAlign: 'center',
+        fontSize: 20
+    },
+    textInputCPF: {
+        height: 30,
+        width: '60%',
+        borderColor: 'white',
+        borderWidth: 2,
+        backgroundColor: '#EAEAEA',
+        marginVertical: 17,
+        textAlign: 'center',
+        alignSelf: 'center',
+    },
```

Comment:
Isso aqui tá repetido não?

Reply:
Ambos estão com os mesmos estilos mas separados por arquivos diferentes, não sei como seria uma boa prática mas de início separei assim justamente para poder alterar cada tela específica caso seja necessário, se usar um para as duas telas se alterar uma pode acabar ficando ruim a outra, pois alterará as 2 telas juntos. Ai pensei assim se no final estiver finalizado posso adicionar e organizar alguns estilos para o produto final. Era o que pensei, faz sentido? ou já tento organizar agora?